### PR TITLE
Fix oneliner script

### PR DESCRIPTION
**Pull request by thomasdiluccio**
Issues have been spotted in the documentation.

### CODE REVIEW

File: docs/src/languages/php/swoole.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ Unlike PHP-FPM’s stateless operating, Swoole relies on establishing persistent
 
 {{< note >}}
 Swoole requires PHP 7.3+.
+
+Swoole installation script is compatible up to PHP 8.0.
```
Comment (CollierCZ) [inline docs/src/languages/php/swoole.md:16]:
```suggestion
The Swoole installation script is compatible up to PHP 8.0.
```

**Review by CollierCZ — APPROVED** [review]
Looks fine to me, but I don't know the details of the script.

**Comment by trolologuy** [comment]
> Looks fine to me, but I don't know the details of the script.

I added additional context.

File: docs/snippets/swoole.yaml [hunk 2]
```diff
@@ -2,4 +2,4 @@ hooks:
     build: |
         set -e
         ...
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/snippets/swoole.yaml:4]: This isn't a generally used pattern in our docs so I wouldn't be pushing it here. I'd either keep as is to indicate it doesn't work on its own or just remove the ellipsis to allow actual copy & pasting. If there's something that should be there, it should be the case that it's not easily copy & pastable.

**Review by CollierCZ — APPROVED** [review]

**Pull request merged**